nixos/modules/services/mail/public-inbox.nix

```diff
@@ -562,15 +571,22 @@ in
               ls -1 "$inbox" | grep -q '^xap' ||
               ${cfg.package}/bin/public-inbox-index "$inbox"
             done
+
+            # Older versions of the module did not make inboxes group-readable.
+            # chmod -R g+r ${stateDir}/inboxes
```

Comment:
Should this be uncommented or what was your idea?

Reply:
Oops, that must have been from when I was testing if it was necessary.